README.md

```diff
@@ -48,15 +48,10 @@ public decimal GetUserStatistic(IUser user)
 }
 ```
 
-Control whether experiments run on a global level by defining a new laboratory.
+Control whether experiments run on a global level.
```

Comment:
Maybe something like "Provide a global condition that determines whether experiments run or not."

Then in the example, let's put a realistic expectation. For example, maybe you'd want to turn off experiments during peak hours.